src/utils/sitemap_parser.py

```
@@ -0,0 +1,171 @@
+    """Raised when network operations fail."""
+
+    pass
+
+
+class ProcessingError(Exception):
+    """Raised when processing operations fail."""
+
+    pass
+
+
+def parse_sitemap_xml(xml_content: bytes) -> List[str]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)